src/gui/N2Go_Gui.php

```diff
@@ -295,7 +293,7 @@ private function disconnect()
      */
     function resetStyles()
     {
-        $style = $_POST['style'];
+        $style = file_get_contents(plugins_url('/lib/newsletter2go_default.json', __FILE__));
```

Comment:
I am not sure if this will work. each form can be different and forms could have different input fields and customer html tags as well

Reply:
I tested, it works. What it does, it just loads default styling for the form from `newsletter2go_default.json` file, instead of loading in runtime from Javascript file and passing via POST method back to WP.

In both cases it always overwrites custom styling with default, so no actual change is happening, just it happens in a different way.

Reply:
Not sure. I still need to check

Reply:
right! better like that